specificatie/naam.yaml

```diff
@@ -8,6 +8,7 @@ components:
   schemas:
     Aanhef:
       type: string
+      pattern: ^[a-zA-Z0-9À-ÿ .\'\-\']{1,650}?$
```

Comment:
@JohanBoer heb je deze ergens vandaan gekopieerd? Hierin staat twee keer `\'`. (moet dan op de plek waar je deze vandaan hebt gehaald ook worden gecorrigeerd)

Reply:
de . hierin betekent "Any single character". Dat is niet de bedoeling. Hier moet denk ik een \ voor staan

Reply:
@JohanBoer  @fsamwel Met dit pattern ook enkele testen doen kijkend naar tekenset BRP (LO GBA Bijlage II Teletex). Wat doen bv. de Z acute, Z dot en Z caron als deze voorkomen in de naam?

Reply:
> @JohanBoer heb je deze ergens vandaan gekopieerd? Hierin staat twee keer `\'`. (moet dan op de plek waar je deze vandaan hebt gehaald ook worden gecorrigeerd)

Gekopieerd uit de request. Zal daar de 2e \'weghalen en dat toevoegen aan deze PR.